### ISSUE TITLE
Improvement for CmsSearchManager#getIndexSolr(CmsObject cms, Map<String, String[]> params)

### ISSUE THREAD

**Issue by kartobi** [alkacon/opencms-core#264]
Is ```solrs.size() == 1``` here correct? (line 761)
```
if (index == null) {
    // index not found -> try to get the first configured Solr index from the config
    List<CmsSolrIndex> solrs = OpenCms.getSearchManager().getAllSolrIndexes();
    if ((solrs != null) && !solrs.isEmpty() && (solrs.size() == 1)) {
        index = solrs.get(0);
    }
}
```
		
Unlikely that projects exactly have only one index.
Wouldn't ```solrs.size() > 0``` be better?

Or don't I understand the comment "try to get the __first__ configured Solr index" correctly?

**Comment by aKandzior** [comment]
Sorry for being inactive on this issue for so long.

We are evaluating if this still persists in current 9.5.x and may fix this for 9.5.3.

**Comment by gWestenberger** [comment]
The comment in the code is misleading. The parameter map should determine which index is to be used, but if it doesn't, returning the first index would be somewhat arbitrary. Unless there is exactly one Solr index.
